src/components/Header.js

```diff
@@ -73,11 +73,16 @@ export default function Header({ session, user }) {
                 >
                   <span className="absolute -inset-1.5" />
                   <span className="sr-only">GitHub Repo</span>
-                  <FontAwesomeIcon
-                    icon={socialIcon("github")}
-                    className="h-6 w-6"
-                    style={{ height: 25, width: 25 }}
-                  />
+                  {(() => {
+                    const { icon } = socialIcon("github");
+                    return (
+                      <FontAwesomeIcon
+                        icon={icon}
+                        className="h-12 w-12 flex-none rounded-full bg-gray-50 p-2"
+                        style={{ height: 50, width: 50 }}
+                      />
+                    );
+                  })()}
```

Comment:
I don't think the function creation is needed?

Reply:
oops, i didn't consider we can directly access the icon using `icon={socialIcon(data.icon).icon}` instead of creating a wrapper function. i will remove the wrapper functions

Reply:
thank you